docs/source/tutorials/pd_colocated_mooncake_multi_instance.md

```diff
@@ -0,0 +1,300 @@
+# PD-Colocated with Mooncake Multi-Instance
+
+## Getting Started
+
+vLLM-Ascend now supports PD-colocated deployment with Mooncake features.
+This guide provides step-by-step instructions to test these features with
+constrained resources.
+
+Using the Qwen2.5-72B-Instruct model as an example, this guide demonstrates
+how to use vllm-ascend v0.11.0rc2 (with vLLM v0.11.0) on two Atlas 800T A2
+nodes to deploy two vLLM instances. Each instance occupies 4 NPU cards and
+uses PD-colocated deployment.
+
+## Verify Multi-Node Communication Environment
+
+### Physical Layer Requirements
+
+- The two Atlas 800T A2 nodes must be physically interconnected via a RoCE
+  network. Without RoCE interconnection, cross-node KV Cache access
+  performance will be significantly degraded.
+- All NPU cards must communicate properly. Intra-node communication uses HCCS,
+  while inter-node communication uses the RoCE network.
+
+### Verification Process
+
+The following process serves as a reference example. Please modify parameters
+such as IP addresses according to your actual environment.
+
+1. Single Node Verification:
+
+   Execute the following commands sequentially. The results must all be
+   `success` and the status must be `UP`:
+
+   ```bash
+   # Check the remote switch ports
+   for i in {0..7}; do hccn_tool -i $i -lldp -g | grep Ifname; done
+   # Get the link status of the Ethernet ports (UP or DOWN)
+   for i in {0..7}; do hccn_tool -i $i -link -g ; done
+   # Check the network health status
+   for i in {0..7}; do hccn_tool -i $i -net_health -g ; done
+   # View the network detected IP configuration
+   for i in {0..7}; do hccn_tool -i $i -netdetect -g ; done
+   # View gateway configuration
+   for i in {0..7}; do hccn_tool -i $i -gateway -g ; done
+   ```
+
+2. Check NPU Network Configuration:
+
+   Ensure that the hccn.conf file exists in the environment. If using Docker,
+   mount it into the container.
+
+   ```bash
+   cat /etc/hccn.conf
+   ```
+
+3. Get NPU IP Addresses:
+
+   ```bash
+   for i in {0..7}; do hccn_tool -i $i -ip -g; done
+   ```
+
+4. Cross-Node PING Test:
+
+   ```bash
+   # Execute the following command on each node, replacing x.x.x.x
+   # with the target node's NPU card address.
+   for i in {0..7}; do hccn_tool -i $i -ping -g address x.x.x.x; done
+   ```
+
+## Run with Docker
+
+Start a Docker container on each node.
+
+```bash
+# Update the vllm-ascend image
+export IMAGE=quay.io/ascend/vllm-ascend:v0.11.0rc2
```

Comment:
```suggestion
export IMAGE=quay.io/ascend/vllm-ascend:|vllm_ascend_version|
```

This doc will be updated follow https://github.com/vllm-project/vllm-ascend/blob/main/docs/source/conf.py#L72